codewof/programming/content/en/evens-out/question.md

```diff
@@ -0,0 +1,7 @@
+# Evens Out
+
+The following function `evens-out(numbers)` is supposed to take a list of numbers and **return** a new list containing only the odd numbers from the original list.
```

Comment:
The function name should have an underscore like it does in the sample solution :)

Reply:
Done :)